h2o-r/h2o-package/R/models.R

```diff
@@ -1542,7 +1542,18 @@ h2o.varimp <- function(object) {
   o <- object
   if( is(o, "H2OModel") ) {
     vi <- o@model$variable_importances
-    if( is.null(vi) ) { vi <- object@model$standardized_coefficient_magnitudes }  # no true variable importances, maybe glm coeffs? (return standardized table...)
+    if( is.null(vi) ) { # may be glm
+      tvi <- object@model$standardized_coefficient_magnitudes 
+      maxCoeff <- max(tvi$coefficients)
+      sumCoeff <- sum(tvi$coefficients)
+      scaledCoeff <- tvi$coefficients/maxCoeff
+      percentageC <- tvi$coefficients/sumCoeff
+      variable <- tvi$names
+      relative_importance <- tvi$coefficients
+      scaled_importance <- scaledCoeff
+      percentage <- percentageC
+      vi <- data.frame(variable, relative_importance, scaled_importance, percentage)
+      }  # no true variable importances, maybe glm coeffs? (return standardized table...)
```

Comment:
All these calculations are performed to make sure the varimp for glm contains the exact same field as the other algos.  Namely, they need to have relative_importance, scaled_importance and percentage.